themes/finna2/templates/RecordDriver/SolrEad/result-list.phtml

```diff
@@ -55,9 +55,11 @@ if ($img):
             <?php // The funky comparison below is due to Laminas translator not supporting empty values (see ExtendedIniReader) ?>
             <?php if (!$this->translationEmpty('Archive Repository')): ?><span class="small"><?=$this->transEsc('Archive Repository')?>:</span><?php endif; ?>
             <?=$this->escapeHtml($this->organisationDisplayName($this->driver)) ?>
-            <?php if ($origination = $this->driver->getOrigination()): ?>
-              <br/><span class="archive-label"><?=$this->transEsc('Archive Origination')?>:</span>
-              <a href="<?=$this->url('search-results')?>?lookfor=%22<?=urlencode($origination)?>%22&amp;type=Author"><?=$this->escapeHtml($origination)?></a>
+            <?php $originations = $this->driver->tryMethod('getOriginations', [], $this->driver->getOrigination()); if ($originations): ?>
+                <br/><span class="archive-label"><?=$this->transEsc('Archive Origination')?>:</span>
+              <?php $originations = (array)$originations; $originationCount = count($originations); foreach ($originations as $i => $origination): ?>
+                <a href="<?=$this->url('search-results')?>?lookfor=%22<?=urlencode($origination)?>%22&amp;type=Author"><?=$this->escapeHtml($origination)?></a><?=$i + 1 < $originationCount ? ' ; ' : ''?>
```

Comment:
Lisätään puolipiste ennen linkkiä muille paitsi ensimmäiselle, niin ei tarvitse count():ia.

Reply:
Kun näitä nyt viilataan, niin olisi hyvä käyttää url-helperiä parametrien lisäämiseen ja ajaa tulos escapeHtmlAttr:n kautta. Ks. esim. https://github.com/NatLibFi/NDL-VuFind2/blob/dev/themes/finna2/templates/librarycards/home.phtml#L79